k9mail/src/main/java/com/fsck/k9/ui/ContactBadge.java

```diff
@@ -0,0 +1,323 @@
+/*
+ * Some code from Android's QuickContactBadge
+ *
+ * Copyright (C) 2009 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * modified by madrat- for customize its look
```

Comment:
I think this should read only:

`Modified by K-9 Developers`

I dislike the idea of maintaining per file attribution in the source code (that's what Git is for) and to avoid endless changes every time we modify the file we should not comment why here (nor are we required to).